src/routes/relay/relay.controller.module.ts

```diff
@@ -3,10 +3,11 @@ import { RelayDomainModule } from '@/domain/relay/relay.domain.module';
 import { RelayService } from '@/routes/relay/relay.service';
 import { RelayController } from '@/routes/relay/relay.controller';
 import { RelayLegacyController } from '@/routes/relay/relay.legacy.controller';
+import { HttpErrorFactory } from '@/datasources/errors/http-error-factory';
 
 @Module({
   imports: [RelayDomainModule],
-  providers: [RelayService],
+  providers: [HttpErrorFactory, RelayService],
```

Comment:
If the `httpErrorFactory` is removed from the `RelayService` we can also remove this provider.

Reply:
Removed in 7bd3e3d48d8d989ace3f9a8163817e9662ebb736.